tool/src/java/org/sakaiproject/gradebookng/tool/panels/AddGradeItemPanelContent.java

```diff
@@ -7,9 +7,12 @@
 import org.apache.wicket.model.Model;
 import org.apache.wicket.model.PropertyModel;
 import org.sakaiproject.gradebookng.tool.model.GbAssignmentModel;
+import org.sakaiproject.service.gradebook.shared.CategoryDefinition;
 
 import java.util.Arrays;
 import java.util.List;
+import java.util.ArrayList;
+import java.util.stream.Collectors;
```

Comment:
Thanks!  Yep, was going to pull in some Java 8 stream stuff but the compiler wouldn't let me get away with it.  Will drop them out.